### ISSUE TITLE
Must not be accessed before initialization

### ISSUE THREAD

**Issue by mbardelmeijer** [vdhicts/cyberfusion-cluster-api-client#75]
## Describe the bug

When creating a FPM Pool, the following error is returned:
`Typed property Vdhicts\Cyberfusion\ClusterApi\Models\FpmPool::$unitName must not be accessed before initialization`

The following also happens with Cron's:
`Typed property Vdhicts\Cyberfusion\ClusterApi\Models\Cron::$emailAddress must not be accessed before initialization`

## Affects

For example: FPM pool / Cron

## Reproduction

Create a FPM pool:
```
(new FpmPool())
   ->setName($this->name)
   ->setVersion($this->phpVersion)
   ->setUnixUserId($this->unixUserId)
   ->setMaxChildren(25)
   ->setClusterId($this->clusterId)
```

## Expected behavior

Get a valid ID back.

## Actual behavior

The exception above is thrown.

## Additional information

It seems to be related to that the $unitName variable should only be set for when the response is given, but the `validateRequired` method in the `create` action calls the `toArray` method, which in turn returns the uninitialized value.

For the cron: it seems to be the emailAddress is nullable, and should be set to something like: `private ?string $emailAddress = null;`

**Comment by dvdheiden** [comment]
Thanks for reporting this bug. Will take a look at it!

**Comment by dvdheiden** [comment]
Fixed in v1.32.2.
